src/main/java/greencity/exception/CheckRepeatingValueException.java

```diff
@@ -0,0 +1,10 @@
+package greencity.exception;
+
+public class CheckRepeatingValueException extends RuntimeException {
+    /**
+     * Generated javadoc, must be replaced with real one.
```

Comment:
replace,


and either you writhing "author" and "version" in all classes or in none